proto/elys/tradeshield/tx.proto

```diff
@@ -19,6 +19,7 @@ service Msg {
   rpc DeletePendingPerpetualOrder (MsgDeletePendingPerpetualOrder) returns (MsgDeletePendingPerpetualOrderResponse);
```

Comment:
@cryptokage1996 can we implement `CancelPerpertualOrders` as well and remove `DeletePendingPerpetualOrder`? cc @Wgil